Procfile

```diff
@@ -0,0 +1 @@
+web: gunicorn server:app
```

Comment:
I am not sure what the impact will be of using presumably default values starting gunicorn , the existing one has several parameters

Reply:
The exist command is there to start each service on a different port in NGD, since they're in isolated containers in cloudfoundry they can all start on the same port.